### ISSUE TITLE
multiplayer turn locking

### ISSUE THREAD

**Issue by jaybl** [jaybl/otrio#2]
Currently there is no system in place to prevent other players from "stealing" others turns. Needs a cycling lock to prevent this based on the current player. 

Was just too lazy to fix this yet.

**Comment by jaybl** [comment]
Various other bugs may well be on there, haven't tested this part nearly enough.

**Comment by Christianlee19** [comment]
Tried running it on iOS but just got a blank window w/ big ups liquid Richard bar. Didn’t throw any error messages though

On Jan 7, 2019, at 4:46 PM, Jay Lin <notifications@github.com<mailto:notifications@github.com>> wrote:


Assigned #2<https://github.com/jaybl/otrio/issues/2> to @Christianlee19<https://github.com/Christianlee19>.

**Comment by jaybl** [comment]
This is correct, you need to press on the menu near the top to start anything

**Comment by Christianlee19** [comment]
> Currently there is no system in place to prevent other players from "stealing" others turns. Needs a cycling lock to prevent this based on the current player.
> 
> Was just too lazy to fix this yet.
